integration/simulation/transaction_manager.go

```diff
@@ -1 +1,97 @@
 package simulation
+
+import (
+	"sync"
+	"time"
+
+	common2 "github.com/obscuronet/obscuro-playground/go/obscuronode/common"
+
+	"github.com/obscuronet/obscuro-playground/go/obscuronode/enclave"
+
+	"github.com/google/uuid"
+	"github.com/obscuronet/obscuro-playground/go/common"
+	wallet_mock "github.com/obscuronet/obscuro-playground/integration/walletmock"
+)
+
+type TransactionManager struct {
```

Comment:
the name of this is a bit inconsistent. Sometimes it's transaction generator sometimes manager.